docs/_sass/_docs-main-content.scss

```diff
@@ -22,7 +22,7 @@
         margin: 10px 0 10px 0;
     }
 
-    ul{
+    > ul {
         @media screen and (max-width: 950px) {
             padding-left: 0;
             margin-left: 3rem;
```

Comment:
I don't think we should style ul in the documentation. In my opinion this should be removed and if the documentation page needs styling for ul it should rely on a specific class

Reply:
fixed